lighthouse-core/report/html/report-styles.css

```diff
@@ -269,7 +269,7 @@
   --header-padding: 16px 0 16px 0;
   --plugin-icon-size: 75%;
   --pwa-icon-margin: 0 7px 0 -3px;
-  --score-container-width: 92px;
+  --score-container-width: 97px;
```

Comment:
what's the effect here/why necessary?

Reply:
the text for the gauge titles was wrapping for Perf/A11y (in DevTools only) b/c the scroll bar steals some width.